vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -138,13 +138,23 @@ public static VimeoClient getInstance() {
 
     public static void initialize(@NotNull Configuration configuration) {
         if (sSharedInstance != null) {
-            sSharedInstance.mConfiguration = configuration;
+            sSharedInstance.mConfiguration.deleteAccount(sSharedInstance.mVimeoAccount);
+            sSharedInstance.setVimeoAccount(null);
```

Comment:
Is this needed since `setVimeoAccount(vimeoAccount)` is done in `setup`?

Reply:
I removed it.